webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/RegistrationCommandsPageConstants.js

```diff
@@ -0,0 +1,10 @@
+import URI from 'urijs';
+import { foremanUrl } from '../../../../foreman_tools';
+
+export const docUrl = (foremanVersion) => {
+  const rootUrl = `https://docs.theforeman.org/${foremanVersion}/`
+  const section = 'Managing_Hosts/index-foreman-el.html#registering-a-host_managing-hosts'
```

Comment:
The link is changing. Is this still safe to cherry pick to 2.5 or would it be a slightly different cherry pick?

Reply:
You are right, this can't be cherry picked to 2.5. But it is not a problem, downstream branch is not rebased anymore over the Foreman 2.5-stable, so there is no need for cherry-pick.

Reply:
But for users of 2.5 it is weird because that documentation link is broken. So I'd argue it still needs a cherry pick.

Reply:
For 2.5 doc link is the old one:
https://docs.theforeman.org/2.5/Managing_Hosts/index-foreman-el.html#registering-a-host-to-project-using-the-global-registration-template_managing-hosts

so no need to cherry-pick it

Reply:
@stejskalleos,  We still need to use the links controller in 2.5 for downstream branding

Reply:
@ShimShtein yop, for downstream 2.5 branch we will cherry-pick it